plugins/arStorageServicePlugin/lib/arStorageServiceUtils.class.php

```diff
@@ -63,6 +63,7 @@ public static function getFileFromStorageService($url)
         $status = curl_getinfo($ch, CURLINFO_RESPONSE_CODE);
 
         curl_close($ch);
```

Comment:
`curl_close()` can be removed as it no longer does anything. See PHP 8 breaking changes [here](https://www.php.net/manual/en/migration80.incompatible.php#migration80.incompatible.resource2object) - search for curl.